recipes/biogeme/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "biogeme" %}
+{% set version = "3.2.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biogeme-{{ version }}.tar.gz
+  sha256: aa3aaa4d32e37fd98b4b23d0630259e07d4645e7ce42c28d57a96c1b4297acc7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools >=63.4.1
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - pandas >=1.3.5
+    - scipy >=1.7.3
+    - tqdm >=4.64.1
+    - tomlkit >=0.11.5
+    - cythonbiogeme ==1.0.2
```

Comment:
This is not present on conda forge, afaik. You should include a recipe for it in this PR also.